src/datachain/lib/file.py

```diff
@@ -206,6 +207,19 @@ def upload(
         file._set_stream(catalog)
         return file
 
+    @classmethod
+    def from_local_path(cls, path: str, catalog: Optional["Catalog"] = None) -> "File":
```

Comment:
what does this method do? why do we need it?

Reply:
This method returns `File` object from local file. We need it to DRY when splitting video into frames/fragments (see examples). It is for local use only, since we support local fs as a storage, in real life we will use `upload` method.

Reply:
can we use `upload` for the local storage as well? (it should be the same to my mind, same code that works in local and for storage)

Reply:
🤔 Let me check 👍